color/src/parse.rs

```diff
@@ -317,7 +317,7 @@ impl<'a> Parser<'a> {
                 let dim_lc = make_lowercase(dim, &mut buf);
                 let scale = match dim_lc {
                     "deg" => 1.0,
-                    "rad" => 180.0 / f64::consts::PI,
+                    "rad" => 1_f64.to_degrees(),
```

Comment:
How about:
```suggestion
                    "rad" => const { 1_f64.to_degrees(), }
```

Reply:
Good idea, will do tomorrow. I do hope it's redundant in practice!